src/dashboard/components/options/OptionsHome.js

```diff
@@ -139,34 +141,37 @@ export default function OptionsHome(props) {
             </Button>
           </p>
         </div>
-        <hr />
-        <h1 className="secondary-text">{t('Coming Soon!')}</h1>
-        <div className="dashboard__options__item disabled">
-          <h4>{t('Auto-Save')}</h4>
-          <Switch
-            disabled
-            isOn={!!settings.user.autoSave || true}
-            handleToggle={() => saveSetting('user.autoSave', !settings.user.autoSave)}
-            labelText={t('By default, use auto-save for projects')}
-          />
-        </div>
-        <div className="dashboard__options__item disabled">
+        <div className="dashboard__options__item">
           <h4>{t('Backup Location')}</h4>
           <FormGroup controlId="backupLocation">
             <ControlLabel>{t('Folder where backups are stored')}</ControlLabel>
-            <FormControl type="text" value={backupLocation} onChange={() => {}} disabled />
+            <p>{backupLocation}</p>
+            <Button bsSize="small" onClick={onChangeBackupLocation}>
+              {t('Choose...')}
+            </Button>
           </FormGroup>
         </div>
-        <div className="dashboard__options__item disabled">
+        <div className="dashboard__options__item">
           <h4>{t('Days of Backup')}</h4>
           <FormGroup controlId="backupDays">
-            <select onChange={(event) => setBackupType(event.target.value)} disabled>
+            <select onChange={(event) => setBackupType(event.target.value)}>
               <option value="days">{t('Days of Backups')}</option>
               <option value="number">{t('Number of Backups')}</option>
             </select>
             {displayBackupOption()}
           </FormGroup>
         </div>
+        <hr />
+        <h1 className="secondary-text">{t('Coming Soon!')}</h1>
+        <div className="dashboard__options__item disabled">
+          <h4>{t('Auto-Save')}</h4>
+          <Switch
+            disabled
+            isOn={!!settings.user.autoSave || true}
```

Comment:
I think `settings.user.autoSave` will be enough

Reply:
I didn't actually write any of that section, it moved down when I re-enabled the other parts. That's still waiting for activation another day looks like :)